test/irmin-pack/dune

```diff
@@ -26,3 +26,8 @@
  (name      common)
  (modules   common)
  (libraries irmin-test irmin-pack))
+
+(library
```

Comment:
Why create a new library here rather than adding a module to `test_pack`?

Reply:
Not a library anymore, thanks !